docs/source/tools/ctc_segmentation.rst

```diff
@@ -0,0 +1,20 @@
+Dataset creation tool based on CTC-Segmentation
+===============================================
+
+This tool provides functionality to align long audio files and the corresponding transcripts into shorter fragments
+that are suitable for an Automatic Speech Recognition (ASR) model training.
+
+More details could be found in `NeMo/tutorials/tools/CTC_Segmentation_Tutorial.ipynb <https://github.com/NVIDIA/NeMo/blob/main/tutorials/tools/CTC_Segmentation_Tutorial.ipynb>`__ (can be executed with `Google's Colab <https://colab.research.google.com/notebooks/intro.ipynb>`_).
+
+The tool is based on the `CTC Segmentation <https://github.com/lumaku/ctc-segmentation>`__ package and
+`CTC-Segmentation of Large Corpora for German End-to-end Speech Recognition
+<https://arxiv.org/abs/2007.09127>`__ :cite:`tools-kurzinger2020ctc`
+
```

Comment:
Do you plan to extend the docs here? You may copy some basic stuff and command line execution from the tutorial here?

Reply:
I'd rather leave the details for the tutorial.